lookupapi/views.py

```diff
@@ -94,6 +93,21 @@ def list(self, request):
         }, context={'request': request}).data)
 
 
+def getPerson_or_404(scheme, identifier, fetch):
+    """
+    Gets a lookup entry for a Person with the given schema and identifier. If the person
+    scheme is "mock" then test user mug99 is returned
```

Comment:
Surely mug99 should only be returned if "mug99" is the identifier? Otherwise GET http://.../people/mock/foobar will return mug99. Also "mug99" is a CRSid so is not part of the mock scheme?

I'd prefer users in the mock scheme to be test0001 to test0500 to harmonise with demo.raven. So mock:test0001 to mock:test0500 exist and return people objects and mock:{anything else} returns a 404.

Reply:
I have mocked now only test0001 to test0500, leaving user-ids test0401 to test0500 as not staff/student, following conventions from raven: https://raven.cam.ac.uk/project/test-demo/

The lookup is still hit and mug99 data retrieved but then is later replaced to the corresponding one based on the API call (identifier is replace). I believe this is more realistic as it hits lookup, and mug99 is a real test than we can use, so it will be more accurate than the one we can create.

Reply:
Sounds sensible. We can always add overrides for specific users should we ever need them.